8.observability/python/app.py

```diff
@@ -0,0 +1,27 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+import flask
+from flask import request, jsonify
+from flask_cors import CORS
+import math
+import sys
+import time
+
+app = flask.Flask(__name__)
+CORS(app)
+
+@app.route('/multiply', methods=['POST'])
+def multiply():
+    ######################################################
```

Comment:
I think the actual python file under distributed calculator can be updated to do this conditionally on the presence of an enviornment var. That env var will be set by the deploy yaml of the python app in this sample.
If you think that would complicate things for the users, I am fine leaving it this way.

Reply:
I think it's more clean to leave the original file as is. I have a feeling the distributed calculator app will be used in many ways and I'd rather not add potential "bugs" to it